src/modules/healtcheck/healtcheck.controller.ts

```diff
@@ -0,0 +1,76 @@
+import { Controller, Get, Version } from '@nestjs/common';
+import {
+  ApiBearerAuth,
+  ApiOperation,
+  ApiResponse,
+  ApiTags,
+} from '@nestjs/swagger';
+import { HealtcheckService } from './healtcheck.service';
+
+@ApiBearerAuth()
+@ApiTags('healtcheck')
+//@Controller('healtcheck')
+@Controller('')
+export class HealtcheckController {
+  constructor(private readonly healtcheckService: HealtcheckService) {}
+
+  @Get('/live')
+  @Version('1')
```

Comment:
this may not be required - the server should (in theory?) be configured already to add version `1` by default to the URL. we can use the `Version` decorator to override it.
See `main.ts` file - the `enableVersioning` code

Reply:
yes, @Version was not required.